src/geometry/geometry.js

```diff
@@ -45,13 +45,15 @@ export default class Geometry {
     if (attributes) {
       this.setAttributes(attributes);
     } else {
-      log.deprecated('inline attributes', 'attributes parameter');
       // TODO this is deprecated
       delete opts.id;
       delete opts.drawMode;
       delete opts.vertexCount;
       delete opts.attributes;
-      this.setAttributes(opts);
+      if (Object.keys(opts).length > 0) {
+        log.deprecated('inline attributes', 'attributes parameter');
```

Comment:
This log is being fired even when a `Geometry` object created without any inline attributes. So wrap it under this if condition to reduce noise.

For example : `new Geometry({drawMode: GL.TRIANGLES}),` would log deprecation warning.

Reply:
Looking at this again.
* I think deleting fields from opts could modify the application passed parameter and should be avoided. 
* I think we should rewrite this, maybe a for-in loop that checks for arrays or typed arrays `ArrayBuffer.isView`, or just drop the warning. 
Or we could add back support for inline attributes, by looping over all attributes and checking for `ArrayBuffer.isView` :)

Reply:
* Removed code to modify `opts`, extract all arrays from opts and pass it to `setAttributes`
* I still prefer deprecating inline attributes, grouping all attributes under `attribute` key is more readable and doesn't conflict if plan to add any other arrays to the `opts`.